src/OrbitGl/App.cpp

```diff
@@ -1435,6 +1437,7 @@ void OrbitApp::StartCapture() {
 
   absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions_map;
   absl::flat_hash_set<uint64_t> frame_track_function_ids;
+  bool record_user_stack_on_wine_syscall_dispatcher = false;
```

Comment:
The idea is that once we have the "collection of the user_stack solution"; this flag will be switched and we do either or.

Reply:
I was also thinking about the Capture Options with a drop down / multiple choice menu. I think in the end, this is the way to go.

However, would it be fine for you to start with this flag and add the UI later (with the potential caveat of not being able to no stop unwinding)? Or would you rather like to add the tri-state here?

Reply:
> However, would it be fine for you to start with this flag and add the UI later (with the potential caveat of not being able to no stop unwinding)? Or would you rather like to add the tri-state here?

I was not talking about adding the UI now, but I was thinking that if we already know that we need an enum, we can add it right away. But if we are not sure, the boolean is fine, and it's still easy to change afterwards.

Reply:
Sounds good.